### PR TITLE
 Update installation instructions for Docker,Node JS  and Docker Compose

### PR DESCRIPTION
**Pull request by IdrisKulubi**
* Added instructions for downloading Docker Compose binary from GitHub repository
* Included commands for verifying installations of Node.js, Docker, and Docker Compose

This commit improves the clarity and completeness of the installation section in the documentation by providing detailed instructions and verification steps for Docker Compose installation.

**Pull Request Requirements:**

* Provide a brief description of the changeset.
* Include a screenshots if applicable
* Ensure that the changeset adheres to the [DCO guidelines](https://github.com/apps/dco).

<sub><a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NjBhOTNiNmU1MGZjYjI5NThiMDRmMGUiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.LcdXbQsdMFAEN6R4eRwVtQxkuZVXlfjYMY7EAjaBzFo">Huly&reg;: <b>UBERF-6252</b></a></sub>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,61 @@ This project offers a convenient method to host Huly using `docker`, designed fo
   - [Docker Compose](https://docs.docker.com/compose/install/)
 - Make sure what docker and `docker compose` commands are available in your terminal (e.g. `docker --version` and `docker compose --version`).
 - Make sure what docker and `docker compose` commands can be executed without sudo (e.g. `docker run hello-world` and `docker compose --version`).
+## Installing Node.js:
+Node.js is a JavaScript runtime that allows you to run JavaScript code outside of a web browser.
+
+
+### Via Package Manager:
```
Comment (aonnikov) [inline README.md]: I would recommend installing it via `nvm`: https://nodejs.org/en/download/package-manager
We have `.nvmrc` in the project so this approach guarantees the required Node.js version will be installed.

File: README.md [hunk 2]
```diff
@@ -45,6 +45,61 @@ This project offers a convenient method to host Huly using `docker`, designed fo
...
+## On Linux:
+Follow the instructions provided in the [Docker documentation](https://docs.docker.com/engine/install/) for installing Docker Engine on your Linux distribution.
+
+## On macOS:
+Download and install Docker Desktop for Mac from the [Docker Hub](https://hub.docker.com/editions/community/docker-ce-desktop-mac).
+
+## On Windows:
+Download and install Docker Desktop for Windows from the [Docker Hub](https://hub.docker.com/editions/community/docker-ce-desktop-windows).
+
+## Installing Docker Compose:
```
Comment (aonnikov) [inline README.md:72]: Please mark as Linux-only step. For MacOS and Windows, Docker Compose is included as part of Docker Desktop so no additional steps are required.

File: README.md [hunk 3]
```diff
@@ -45,6 +45,61 @@ This project offers a convenient method to host Huly using `docker`, designed fo
...
+Docker Compose is a tool for defining and running multi-container Docker applications.
+
+### Using Python Package Manager (pip):
+```bash
+sudo pip install docker-compose
```
Comment (aonnikov) [inline README.md]: Please remove this. We can't list this as a recommended approach of docker-compose installation. PyPI version of docker-compose is legacy and is not maintained anymore.
https://github.com/docker/compose/tree/v1?tab=readme-ov-file#warning-compose-v1-is-deprecated-warning

File: README.md (hunk 2 above)
Comment (IdrisKulubi) [reply README.md:72]: Thank you for your input! I've updated the README.md to clarify the installation steps for Docker Compose.,Installing using nvm  as per your suggestion,I've marked the installation of Docker Compose as a Linux-only step   and  removed the using of  python package Manager to install docker-compose

Please feel free to review the changes and let me know if there's anything else I should add or modify.

**Comment by aonnikov** [comment]
Closing as we don't want to maintain installation instructions for 3d party products in our repository.
We have links to them and that should be enough for most of users.

**Pull request closed without merging**